modules/building/pages/apply-task-migrations.adoc

```diff
@@ -0,0 +1,113 @@
+= Applying task migrations
+
+{ProductName} allows task owners to write migrations to bring pipeline changes
+to users. Generally, users do not need to care about how the migrations are
+defined and how {ProductName} applies migrations to their build pipelines.
+MintMaker applies the migrations automatically as part of the regular pull requests that update Tekton tasks. However, in case users requires to apply
```

Comment:
```suggestion
MintMaker applies the migrations automatically as part of the regular pull requests that update Tekton tasks. However, in case users are required to apply
```